central/values.yaml

```diff
@@ -61,7 +61,7 @@ centralledger:
       enabled: false
       image:
         repository: mojaloop/event-sidecar
-        tag: v8.7.3
+        tag: v8.8.0-snapshot.3
```

Comment:
```suggestion
        tag: v8.7.3
```

Reply:
The snapshot versions are perf changes, so please do not include them until they're merged into master and then released